app/test_split_integer.py

```diff
@@ -2,20 +2,33 @@
 
 
 def test_sum_or_the_parts_should_be_equal_to_value():
-    pass
+    value = 10
+    number_of_parts = 2
+    assert sum(split_integer(value, number_of_parts)) == value
 
 
 def test_should_split_into_equal_parts_when_value_is_divisible_by_parts():
-    pass
+    value = 10
+    number_of_parts = 2
+    assert split_integer(value, number_of_parts). \
+        count(value / number_of_parts) == \
+           number_of_parts
```

Comment:
Try not to use `\` for the line continuation.
```suggestion
assert split_integer(value, number_of_parts).count(
    value / number_of_parts
) == number_of_parts
```